iphone/Maps/UI/PlacePage/PlacePage.storyboard

```diff
@@ -510,7 +530,7 @@
                                 <rect key="frame" x="16" y="16" width="343" height="224"/>
                                 <subviews>
                                     <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="RUc-Qr-8Sb">
-                                        <rect key="frame" x="0.0" y="0.0" width="343" height="164"/>
+                                        <rect key="frame" x="0.0" y="0.0" width="343" height="44"/>
```

Comment:
nit: it would be great to skip non-relevant changes, or drop them completely.

Reply:
This change (and some others) happened because I resolved some constraint conflicts which was detected by xcode in storyboard (marked as red dots): 164 is incorrect
<img width="1722" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/6c715052-b050-4796-8fef-9334d36bed4e">

Reply:
Ok, hope it didn't break something )